input/pagecontent/medicine-information.md

```diff
@@ -14,7 +14,7 @@ It is anticipated that future releases of AU Core will define AU Core profiles o
 - MedicationDispense (with AU Core Medication) are used to support dispense records and medication management use cases.
 - MedicationStatement (with AU Core Medication) are used to support summary statements of medicine use.
 
-**Medicinal Product Identification**
+### Medicinal Product Identification
 
 For extemporaneous medications, it is expected the medication code is the primary mechanism to identify a medicine. In this case, a text only list of ingredients may be supplied or may be coded using a medicines terminology.
 
```

Comment:
Below "The guidance for how to support coded or text is summarised below:"
A new first point should actually be something around stating that brand name, generic name, manufacturer, item form and strength can be understood from supplying a single code in Medication.code if the code contains that information

Currently the guidance implies that a system must supply multiple codings in order to convey that information and this isn't the case.

Reply:
1. @dt-r was your intention for this information to be the first point under '1. For coded support for brand name...'? 
Given manufacturer is not represented in AMT or PBS codes, would it be appropriate for this point to say something like "Brand or generic name, item form and strength can be understood from supplying a single code in Medication.code if the code contains that information"?


2. Amended, and removed duplicate 'extension' from 'Medication Type extension extension...'
3. Amended.
4. Amended.

Reply:
1. The intention is for it to **be the first point**. Not the first point under something else. And yes I understand what you are saying, please craft the initial point taking all of that into account. Fully coded all parts does require additional things, which is what the next 2 points do offering both a coded and a non-coded way of doing things.

Reply:
Noted and amended.

Reply:
1. the text line isn't guidance on what and how a code can work to provide parts of information on branding etc. and be capable of being a single code that when resolved may be capable of carrying or finding information on strength, branding, form etc. What needs to be added is guidance on medicinal product identification and the capability to do so through a single code depending on the terminology. The audience are readers who don't know what medicine coding can do.

To become familiar with the capabilities of SNOMED and AMT, I would recommend a discussion with Dion or Matt Cordell.